### ISSUE TITLE
wolf does not output in real-time

### ISSUE THREAD

**Issue by Almenon** [Duroktar/Wolf#35]
Wolf outputs everything when the program is finished.  It would be nice for long-running programs or programs that use sleep for it to output in real-time.

Unfortunately this would require major refactoring based on your comment in #31

**Comment by Duroktar** [comment]
Yes, this is in the same boat as #31. I've been busy at work lately and have let these issues build up a bit. I guess I missed this one.

Anyways, I'll leave it open under FAQ so people can follow the links if they have similar issues or questions.

Cheers
